testsuite/MDAnalysisTests/utils/test_log.py

```diff
@@ -147,24 +149,32 @@ class TestSetVerbose(object):
 
     @pytest.mark.parametrize('verbose, quiet, default, result', [
         (True, False, True, True),  # Everything agrees verbose should be True
-        (False, True, False, False),  # Everything agrees verbose should be False
-        (True, False, False, True),  # Make sure the default does not overwrite the user choice
-        (False, True, True, False),  # Make sure the default does not overwrite the user choice
+        (False, True, False, False),# Everything agrees verbose should be False
+        (True, False, False, True), # Make sure the default does not overwrite the user choice
+        (False, True, True, False), # Make sure the default does not overwrite the user choice
+        (None, True, False, False), # Verbose is not provided
+        (None, False, False, True), # Verbose is not provided
+    ])
+    def test__set_verbose_deprecated(self, verbose, quiet, default, result):
+        with pytest.deprecated_call():
+            assert _set_verbose(verbose=verbose, quiet=quiet, default=default) == result
+
+    @pytest.mark.parametrize('verbose, quiet, default, result', [
         (True, None, False, True),  # Quiet is not provided
-        (False, None, False, False),  # Quiet is not provided
-        (None, True, False, False),  # Verbose is not provided
-        (None, False, False, True),  # Verbose is not provided
-        (None, None, True, True),  # Nothing is provided
-        (None, None, False, False),  # Nothing is provided
+        (False, None, False, False),# Quiet is not provided
+        (None, None, True, True),   # Nothing is provided
+        (None, None, False, False), # Nothing is provided
     ])
     def test__set_verbose(self, verbose, quiet, default, result):
         assert _set_verbose(verbose=verbose, quiet=quiet, default=default) == result
 
     @pytest.mark.parametrize('verbose', (True, False))
     def test__set_verbose_invalid_args(self, verbose):
-        with pytest.raises(ValueError):
-            # setting quiet==verbose is a contradiction
-            _set_verbose(verbose=verbose, quiet=verbose, default=None)
+        # can't combine the two context managers
+        with pytest.deprecated_call():
```

Comment:
Is a shortcut for `pytest.warns(DeprecationWarning)`. It basically says that the statements in the context have to raise a DeprecationWarning